### ISSUE TITLE
REPO Administration

### ISSUE THREAD

**Issue by aethelwulffe** [LibreHealthIO/lh-ehr#1407]
Hello all,
I would like to discuss elevation of privileges of a couple of long-time contributors to Repo level administration. 

A number of contributors have held to a steady involvement here for a good while.  They have shown themselves capable, and present themselves as good communicators and collaborators.  Overall experience with the project is climbing, they understand a great many issues, and they may be getting exposed to actual clinical-side experience in a number of ways.
  We currently have a dearth of integrators/managers here.  I feel now is the time to elevate some of our heavy-hitters and dump more responsibility on them.

Any opinions/nominations here?

**Comment by muarachmann** [comment]
Hi Art, I for one have loved the goals of EHR and have been thinking a lot
about it lately. I am a final year student as you all should know and will
round up with my B.Eng this year hopefully. Librehealth in general has been
welcoming for me and would live to make my stay here. Not only did it
teach/introduce me to open source but I felt a sense of improvements in my
skills all round. A big thanks to Art, Robby, Terry, Tony, Htuck , trodige
, konikodes , and our doc intern Virginia and the rest of the EHR. That's
said I think I will be willing to get some responsibilities if accorded.

**Comment by aethelwulffe** [comment]
Yes, you are on my list.  Naveen and Rodrigue have write permissions
already.  Just wanted you to ask.  The big point is that you have been
doing them already without being asked.

We need more involvement in the overall project management, which means
"where do we go next".
